node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -66,6 +66,8 @@ const LRU_OBSERVED_BLOCKS_CAPACITY: NonZeroUsize = match NonZeroUsize::new(20) {
 pub struct ChainScraper {
 	/// All candidates we have seen included, which not yet have been finalized.
 	included_candidates: HashSet<CandidateHash>,
+	/// All candidates we have seen backed
+	backed_candidates: HashSet<CandidateHash>,
 	/// including block -> `CandidateHash`
 	///
 	/// We need this to clean up `included_candidates` on finalization.
```

Comment:
It does not look like backed_candidates get ever cleaned up.

Reply:
Fixed